lib/jazzy/podspec_documenter.rb

```diff
@@ -116,6 +116,7 @@ def podfile
                  deterministic_uuids: false
 
         [podspec, *podspec.recursive_subspecs].each do |ss|
+          next if ss.test_specification?
```

Comment:
this will fail for old cocoapods versions that don't have the method

Reply:
Oh 🤦‍♂️ yes of course.  Thanks.  You think better to `responds_to?` or require cocoapods 1.3 in the gemspec?

Reply:
probably better to add a `responds_to?` for now?